gauges/logical_replication.go

```diff
@@ -0,0 +1,95 @@
+package gauges
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/ContaAzul/postgresql_exporter/postgres"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type slots struct {
+	SlotName     string  `db:"slot_name"`
+	IsSlotActive float64 `db:"active"`
```

Comment:
```suggestion
	Active float64 `db:"active"`
```
The name of the struct is already `slots`, so `slots.Active` is better than `slots.IsSlotActive`